infrastructure/configuration/devops-agents/config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+APT_REPOSITORIES=(
+  "main"
+  "restricted"
+  "universe"
+  "multiverse"
+)
+
+COMMON_PACKAGES=(
```

Comment:
Do we need the build-essential or was it put here from ODT and its docker build, or does .NET need it?